pkg/memory/memory.go

```diff
@@ -29,6 +29,15 @@ type Module struct {
 	Vendor       string `json:"vendor"`
 }
 
+// HugePage describes huge page info
+type HugePage struct {
```

Comment:
Let's call this struct `HugePageAmounts` since it's not describing the huge page itself but rather a set of distinct finite amounts of huge pages of a particular size within a memory area.

Reply:
fixed